index.html

```diff
@@ -8,7 +8,178 @@
     <title>PricingPage</title>
     <link rel="preconnect" href="https://fonts.googleapis.com" />
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
-    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@500;600;700&display=swap"
-      rel="stylesheet"/>
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@500;600;700&display=swap"
+      rel="stylesheet"
+    />
   </head>
+
+  <body>
+    <header>
+      <button class="btn-back">
+        <img class="img" src="img/Arrow Forward.svg" />Back
+      </button>
+      <h1 class="title">Pricing</h1>
+      <div class="subtitle">You can update or downgrage at any time</div>
+    </header>
+
+    <div class="switch-container">
+      <span class="monthly">Monthly</span>
+      <img class="img-switch" src="/img/Switch.svg" />
+      <span class="annually">Annually</span>
+      <span class="save">(save 10%)</span>
+    </div>
+
+    <main class="main">
+      <!-- block 1 -->
+      <div class="plan">
+        <div class="block-header">
+          <div class="avatar1"></div>
+
+          <div class="header-text">
+            <div class="plan-name">Pro</div>
+            <div class="price">
+              <span class="dollar">$</span>
+              <span class="number">7</span>
+              <span class="month">Month</span>
+            </div>
+          </div>
+        </div>
+        <div class="info-container">
+          <img class="note" src="/img/Music Note.svg" alt="song-icon" />
+          <span class="songs">50 songs</span>
+          <img src="/img/Storage.svg" alt="" class="storage" />
+          <span class="memory">2 Gb</span>
+        </div>
+
+        <div class="line"></div>
+
+        <ul class="list">
+          <li class="list-item">
+            <img class="chek-box" src="/img/check_big.svg" />Access to pro jobss
+          </li>
+          <li class="list-item">
+            <img class="chek-box" src="/img/check_big.svg" />Free Communication
```

Comment:
i recommend you to make that cases in two lines, it will be easy to read
```suggestion
            <img class="chek-box" src="/img/check_big.svg" />
            Free Communication
```